scripts/README.md

```diff
@@ -46,4 +46,4 @@ The `lancache.conf` should be copied into the `/etc/dnsmasq.d/` location but als
 You can copy the `*.hosts` file to any location other than `/etc/dnsmasq.d/` as this location is utilised only for `*.conf` files.
```

Comment:
`Any location other than /etc/dnsmasq/hosts/`

Reply:
That would be wrong though, `/etc/dnsmasq.d/` is where `.conf` files are intended to go and if a `.hosts` file is loaded in that directory dnsmasq will fail to start.

Reply:
Yup, I see. Misread the intent